odbc/src/api/col_attr_describe_tests.rs

```diff
@@ -8,17 +8,30 @@ use std::sync::RwLock;
 
 mod unit {
```

Comment:
this file is mostly just updates to provide an env and conn handle so we don't panic